tests/legacy/legacy_literal.json

```diff
@@ -0,0 +1,56 @@
+[
+    {
+        "given": {
+            "foo": [{"name": "a"}, {"name": "b"}],
+            "bar": {"baz": "qux"}
+        },
+        "cases": [
+            {
+                "expression": "`foo`",
+                "result": "foo"
+            },
+            {
+                "comment": "Double quotes must be escaped.",
+                "expression": "`foo\\\"quote`",
+                "result": "foo\"quote"
+            },
+            {
+                "expression": "`✓`",
+                "result": "✓"
+            },
+            {
+                "comment": "Double quote in literal",
+                "expression": "`foo\\\"bar`",
+                "result": "foo\"bar"
```

Comment:
Isn't this test redundant with "Double quotes must be escaped."?

Reply:
Well it technically is. However, as the purpose of JEP-12 focused on the usage of mandatory double-quotes for surrounding test, I figure it does no harm to keep this text.

Reply:
I mean it's literally not testing anything interesting that isn't completely covered; the only difference is `quote` → `bar`.
```diff
-        "expression": "`foo\\\"quote`",
-        "result": "foo\"quote"
+        "expression": "`foo\\\"bar`",
+        "result": "foo\"bar"
```